fs/operations.go

```diff
@@ -19,6 +20,11 @@ import (
 	"golang.org/x/text/unicode/norm"
 )
 
+type jsonPrinter struct {
```

Comment:
I would think you should put this with the method definitions - go doesn't need it to be before its use and it looks a bit out of place here... A comment as to what it is for would be great.